src/BiometryService/BiometryType.cs

```diff
@@ -0,0 +1,25 @@
+namespace BiometryService
+{
+	/// <summary>
+	///		Defines the different types of biometric identifiers.
+	/// </summary>
+	public enum BiometryType
+	{
+		/// <summary>
+		///		No biometric identifier.
+		/// </summary>
+		None = 0,
+		/// <summary>
+		///		Fingerprint identifier.
+		/// </summary>
+		Fingerprint,
+		/// <summary>
+		///		Face identifier.
+		/// </summary>
+		Face,
+		/// <summary>
+		///		Managed by the OS
+		/// </summary>
+		FaceOrFingerprint
```

Comment:
will this bite us in the butt later? (if for example iris scan comes), we could use enum flags instead?